lib/plugins/encrypted_parameters/kms_key.rb

```diff
@@ -13,10 +15,24 @@ def initialize(arn)
         def self.create
           resp = Aws::KMS::Client.new.create_key
           arn = resp.key_metadata.arn
-
           new(arn)
         end
 
+        def update
+          standard_tags=get_tags
+          resp = @kms_client.tag_resource({
+            key_id: @arn, # arn of the CMK being tagged
+            tags: standard_tags, # A list of tags.
+          })
+        end
+
+        def get_tags
```

Comment:
Can this be other descriptive name, something like stack_tags instead? and usually get* set* methods are used as accessor methods on the class object itself in ruby.